packages/core/src/routes/experience/classes/verifications/password-verification.ts

```diff
@@ -41,54 +42,65 @@ export class PasswordVerification implements Verification {
       id: generateStandardId(),
       type: VerificationType.Password,
       identifier,
+      verified: false,
     });
   }
 
   readonly type = VerificationType.Password;
   public readonly identifier: InteractionIdentifier;
   public readonly id: string;
-  /* The userId of the user that has been identifier by the given identifier and password */
-  private userId?: string;
+  private verified: boolean;
 
   constructor(
     private readonly libraries: Libraries,
     private readonly queries: Queries,
     data: PasswordVerificationRecordData
   ) {
-    const { id, identifier, userId } = data;
+    const { id, identifier, verified } = data;
 
     this.id = id;
     this.identifier = identifier;
-    this.userId = userId;
+    this.verified = verified;
   }
 
   /** Returns true if a userId is set */
   get isVerified() {
-    return this.userId !== undefined;
-  }
-
-  get verifiedUserId() {
-    return this.userId;
+    return this.verified;
   }
 
   /** Verifies the password and sets the userId */
   async verify(password: string) {
```

Comment:
not sure whether this is a good name, if I ignore the input parameter's name password, I do not know this method only verifies the user's password.

Reply:
synced offline